frontend/public/locales/en/filter-toolbar.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Name": "Name",
+  "Label": "Label",
+  "labels": "labels",
+  "name": "name",
+  "{{value}}": "{{value}}",
```

Comment:
If the only thing here is the variable, this doesn't need to be externalized.

Reply:
Good catch!